packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -158,7 +158,7 @@ const SearchField = forwardRef<HTMLInputElement, SearchFieldProps>(
           autoFocus={hasInitialFocus}
           data-se={testId}
           endAdornment={
-            defaultValue && (
+            (inputValues?.defaultValue || inputValues?.value) && (
```

Comment:
`value` and `defaultValue` are strings. We'd never have `0`

Reply:
They are basically the same operator in this case. That being said, in this case I think `||` makes it more clear that if `defaultValue` OR `value` exists, show this thing. I feel like `??` is used more commonly in cases where you want coercion. Like...
```js
return thisThing ?? orThisThingIfTheLeftHandSideIsFalse
```

Reply:
> The OR operator considers any falsy value (e.g. 0, '', false) to be equivalent to false, while the nullish coalescing operator only considers null and undefined to be nullish.

So, you are right in the event the first value _could_ be `0`, you'd want `??`

Reply:
Great point that it can't ever be `0`. Looking at the component, it's only ever taking string values.

Using `??` makes it clear our logic cares only if the value doesn't exist or is `null`. It's using `!== undefined && `!== null` rather than doing the more ambiguous `!= null`.

I'm not gonna block the PR on this.

Reply:
Actually, I think we DO want `||` here because we don't want `""` to be considered true right?